### PR TITLE
fix: ie11 image

### LINKED ISSUE #206: Images look blurred on IE
related issue https://github.com/orgs/comicrelief/projects/13#card-28886908

### PR DESCRIPTION
**Pull request by LouiaR**
Fixes #206

### CODE REVIEW

**Comment by AndyEPhipps** [comment]
@LouiaR - does this still need finishing/merging?

**Comment by LouiaR** [comment]
It's on pause for now.

File: src/components/Atoms/Picture/Picture.js [hunk 1]
```diff
@@ -34,25 +37,65 @@ const Picture = ({
...
+        {...rest}
+      >
         <Image
           alt={alt}
           height={height}
           width={width}
           objectFit={objectFit}
           data-src={image}
-          className="lazyload"
+          className={`lazyload ${!objFitState && 'no-object-fit'}`}
```
Comment (gusliedke) [inline src/components/Atoms/Picture/Picture.js:79]: It's returning `false` as a class on image.

Comment (AndyEPhipps) [reply src/components/Atoms/Picture/Picture.js:79]: I realised I don't even need that class anymore, so will remove it

**Comment by gusliedke** [comment]
Are we still supporting IE11? It's an aesthetic issue. not sure if it's worthy to add all this complexity and logic to Picture. But the work is done. Louis and I didn't have a chance to discuss about it.

**Comment by AndyEPhipps** [comment]
> Are we still supporting IE11? It's an aesthetic issue. not sure if it's worthy to add all this complexity and logic to Picture. But the work is done. Louis and I didn't have a chance to discuss about it.

I think we are, and it'll support any other non-object-fit legacy browsers too. 

But yeah, maybe not the most useful, but it's done now?

**Comment by gusliedke** [comment]
> > Are we still supporting IE11? It's an aesthetic issue. not sure if it's worthy to add all this complexity and logic to Picture. But the work is done. Louis and I didn't have a chance to discuss about it.
> 
> I think we are, and it'll support any other non-object-fit legacy browsers too.
> 
> But yeah, maybe not the most useful, but it's done now?

Not sure if it's been tested. If Krupa approves it we merge it if it causes other issues I'd hold.

**Comment by AndyEPhipps** [comment]
> > > Are we still supporting IE11? It's an aesthetic issue. not sure if it's worthy to add all this complexity and logic to Picture. But the work is done. Louis and I didn't have a chance to discuss about it.
> > 
> > 
> > I think we are, and it'll support any other non-object-fit legacy browsers too.
> > But yeah, maybe not the most useful, but it's done now?
> 
> Not sure if it's been tested. If Krupa approves it we merge it if it causes other issues I'd hold.

👍  I've just asked Krupa to check it out when she can; I've done a fair bit of testing on IE11 against live and it's all looking fine, but can wait for her verification

**Comment by AndyEPhipps** [comment]
@gusliedke - Krupa's QA'd this now: https://github.com/comicrelief/comicrelief-contentful/pull/776

**Review by gusliedke — APPROVED** [review]

**Pull request merged**